data_reader.py

```diff
@@ -0,0 +1,121 @@
+import csv
```

Comment:
Add doc strings explaining what this module does. Also add docstring for classes and function. Take a look here for example: 
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html